
### **Probability Basics: Events, Experiments, and Sample Space**  

#### **1. Experiment**  
An **experiment** is a process or action that results in an **outcome**.  

#### **Types of Experiments:**  
1. **Deterministic Experiment** – The outcome is certain and predictable.  
   - Example: Boiling water at 100°C (at standard pressure) will always turn into steam.  
2. **Random Experiment** – The outcome is not certain and can vary.  
   - Example: Tossing a coin (it can land on heads or tails).  


#### **2. Sample Space (S)**  
The **sample space** is the set of all possible outcomes of an experiment.  

#### **Example: Tossing a Coin**  
- The sample space is:  
  $$
  S = \{ \text{Heads}, \text{Tails} \}
  $$  

#### **Example: Rolling a Die**  
- The sample space is:  
  $$
  S = \{1, 2, 3, 4, 5, 6\}
  $$  



#### **3. Event (E)**  
An **event** is a subset of the sample space. It represents a specific outcome or a group of outcomes.  

#### **Example: Rolling a Die**  
- Sample Space:  
  $$
  S = \{1, 2, 3, 4, 5, 6\}
  $$  
- Event: Getting an even number  
  $$
  E = \{2, 4, 6\}
  $$  


#### **4. Probability of an Event**  
The probability of an event **E** occurring is given by:  
$$
P(E) = \frac{\text{Number of favorable outcomes}}{\text{Total number of outcomes in the sample space}}
$$  

#### **Example: Rolling a Fair Die**  
- Sample Space: $ S = \{1, 2, 3, 4, 5, 6\} $  
- Probability of rolling an even number:  
  $$
  P(\text{Even number}) = \frac{3}{6} = \frac{1}{2}
  $$  
- Probability of rolling an odd number:  
  $$
  P(\text{Odd number}) = \frac{3}{6} = \frac{1}{2}
  $$  
- Probability of rolling a **20** (which is not in the sample space):  
  $$
  P(20) = 0
  $$  


#### **Valid When?**  
This formula is only valid when **all outcomes are equally likely** to occur.  

#### **Examples of Equally Likely Outcomes:**  
- Rolling a **fair die**: $ S = \{1, 2, 3, 4, 5, 6\} $  
- Flipping a **fair coin**: $ S = \{H, T\} $  



#### **2. Laws of Probability**  

#### **Law 1: Probability Range**  
$$
0 \leq P(E) \leq 1
$$  
- **P(E) = 0** → The event **cannot** happen (impossible).  
  - Example: Rolling a 7 on a standard die.  
- **P(E) = 1** → The event is **certain** (deterministic).  
  - Example: The sun rising tomorrow.  

#### **Law 2: Sum of Probabilities in Sample Space**  
$$
\sum P(\text{outcomes}) = 1
$$  
- The total probability of all possible outcomes in a sample space **must sum to 1**.  

#### **Example: Rolling a Fair Die**  
Possible outcomes: $ S = \{1, 2, 3, 4, 5, 6\} $  
Each outcome has a probability of $ \frac{1}{6} $, so:  
$$
P(1) + P(2) + P(3) + P(4) + P(5) + P(6) = \frac{1}{6} + \frac{1}{6} + \frac{1}{6} + \frac{1}{6} + \frac{1}{6} + \frac{1}{6} = 1
$$  



In [5]:
import random
sample_sizes = [5, 10, 100, 1000, 10000, 100000]
for sample_size in sample_sizes:
    heads_count = 0
    for _ in range(sample_size):
        toss_result = random.randint(0, 1)
        if toss_result == 1:
            heads_count += 1
    prob = heads_count / sample_size
    print(f"Estimated probability of heads using sample size: {sample_size}, probability: {prob}")

Estimated probability of heads using sample size: 5, probability: 0.2
Estimated probability of heads using sample size: 10, probability: 0.3
Estimated probability of heads using sample size: 100, probability: 0.57
Estimated probability of heads using sample size: 1000, probability: 0.503
Estimated probability of heads using sample size: 10000, probability: 0.5075
Estimated probability of heads using sample size: 100000, probability: 0.50083



#### **1. Probability of Impossible & Certain Events**  
- **P(0) = 1** → The probability of a **certain** event occurring is **1**.  
- **P(∅) = 0** → The probability of an **impossible** event is **0**.  

#### **2. Types of Probability**  

1. **Joint Probability (P(A ∩ B))**  
    - The probability of two events **A** and **B** happening together.  
2. **Conditional Probability (P(A | B))**  
    - The probability of event **A** occurring **given that** event **B** has already occurred.  
3. **Marginal Probability (P(A))**  
    - The probability of a single event occurring, without considering any other events.  


#### **Classification of a Standard 52-Card Deck**  

A standard deck of **52 playing cards** is divided into **four suits**, each containing **13 cards**:

#### **1. Suits in a Deck**  
- **Hearts (♥)** → Red  
- **Diamonds (♦)** → Red  
- **Clubs (♣)** → Black  
- **Spades (♠)** → Black  

Each suit has the same **13 ranks**:  
- **Number Cards (1-10)** → (Ace is sometimes considered 1)  
- **Face Cards** → Jack (J), Queen (Q), King (K)  

#### **2. Full Breakdown of the 52 Cards**  

| **Suit**    | **Number Cards (1-10)** | **Face Cards (J, Q, K)** | **Total** |
|||-||
| **Hearts (♥) - Red**    | 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 | J, Q, K | 13 |
| **Diamonds (♦) - Red**  | 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 | J, Q, K | 13 |
| **Clubs (♣) - Black**    | 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 | J, Q, K | 13 |
| **Spades (♠) - Black**   | 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 | J, Q, K | 13 |

✅ **Total Cards = 13 (Hearts) + 13 (Diamonds) + 13 (Clubs) + 13 (Spades) = 52**  

#### **3. Classification by Color**  
- **Red Cards:** Hearts (♥) + Diamonds (♦) → **26 cards**  
- **Black Cards:** Clubs (♣) + Spades (♠) → **26 cards**  

#### **4. Classification by Type**  

| **Type**          | **Cards Included**            | **Total Count** |
||||
| **Number Cards** | 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 in each suit | 40 |
| **Face Cards**   | J, Q, K in each suit         | 12 |
| **Aces**        | 1 per suit (Hearts, Diamonds, Clubs, Spades) | 4 |

#### **5. Classification by Even and Odd Number Cards**  

| **Type**        | **Odd Numbers** (1, 3, 5, 7, 9) | **Even Numbers** (2, 4, 6, 8, 10) |
|-||--|
| **Hearts (♥)**  | 1, 3, 5, 7, 9 (5 cards)       | 2, 4, 6, 8, 10 (5 cards)       |
| **Diamonds (♦)** | 1, 3, 5, 7, 9 (5 cards)       | 2, 4, 6, 8, 10 (5 cards)       |
| **Clubs (♣)**   | 1, 3, 5, 7, 9 (5 cards)       | 2, 4, 6, 8, 10 (5 cards)       |
| **Spades (♠)**  | 1, 3, 5, 7, 9 (5 cards)       | 2, 4, 6, 8, 10 (5 cards)       |
| **Total**      | 20 odd-numbered cards         | 20 even-numbered cards         |

#### **6. Special Categories**  
- **Total Face Cards:** 12 (3 per suit × 4 suits)  
- **Total Aces:** 4 (1 per suit)  
- **Total Red Odd Number Cards:** 10 (5 in Hearts + 5 in Diamonds)  
- **Total Black Odd Number Cards:** 10 (5 in Clubs + 5 in Spades)  


#### **4. Joint Probability Example**  

**Experiment:** Drawing a card from a standard 52-card deck.  
- **Sample Space (SS)**: **52 cards**  
- **Events:**  
  1. **A (Red Card)**: There are **26** red cards.  
  2. **B (Odd Numbered Card)**:  
     - Numbered cards are from **1-10**.  
     - Odd-numbered red cards: **{1, 3, 5, 7, 9}** in **Hearts (♥) and Diamonds (♦)**.  
     - **Total odd-numbered red cards = 5 (♥) + 5 (♦) = 10**.  

#### **Joint Probability Calculation:**  
$$
P(A \cap B) = \frac{\text{Number of Red Odd Cards}}{\text{Total Cards in Deck}}
$$  
$$
P(A \cap B) = \frac{10}{52}
$$  



#### **Joint Probability**

#### **Definition**  
Joint probability refers to the probability of two events occurring **simultaneously**. It is denoted as:  

$$
P(A \cap B) = P(A, B)
$$

where:  
- **A** and **B** are two events.
- **P(A ∩ B)** is the probability of **both** events occurring at the same time.

#### **1. Experiment**  
- **Drawing a single card from a standard deck of 52 cards.**

#### **2. Sample Space (SS)**  
$$
S = 52 \text{ (Total number of cards in the deck)}
$$

#### **3. Events Defined**  
- **A (Shant's Event)** → The card is **red**.  
  - Total red cards (**Hearts + Diamonds**) = **26**  
- **B (Mudit's Event)** → The card is an **even-numbered card**.  
  - Even-numbered cards per suit: **2, 4, 6, 8, 10**  
  - Each suit has **5 even-numbered cards**  
  - Total even-numbered cards in red suits (♥♦):  
    - **Hearts (♥) → {2, 4, 6, 8, 10}** (5 cards)  
    - **Diamonds (♦) → {2, 4, 6, 8, 10}** (5 cards)  
    - **Total even red cards = 5 (♥) + 5 (♦) = 10**  

#### **4. Joint Probability Calculation**  
$$
P(A \cap B) = \frac{\text{Number of Red Even Cards}}{\text{Total Cards in Deck}}
$$  
$$
P(A \cap B) = \frac{10}{52} = \frac{5}{26}
$$

#### **Joint Probability Properties**
1. **Symmetry of Joint Probability**  
   $$
   P(A, B) = P(B, A)
   $$  
   $$
   P(A \cap B) = P(B \cap A)
   $$  
   This means the order of the events does not change the joint probability.

#### **Summary of Values**  
- **Total red cards (A)** = **26**  
- **Total even-numbered cards (B)** = **20**  
- **Total red even-numbered cards (A ∩ B)** = **10**  
- **Joint Probability:**  
  $$
  P(A \cap B) = \frac{10}{52} = \frac{5}{26}
  $$  


#### **Union and Marginal Probabilities in a Deck of Cards**  

#### **1. Understanding the Notation**
- **Shant's Event (A):** Drawing a **red card**  
  - **Total red cards** = **26**  
- **Mudit's Event (B):** Drawing an **even-numbered card**  
  - **Total even-numbered cards** = **20**  

#### **Joint Probability (P(A ∩ B)) → Both A and B occur**  
- **Red Even Cards:**  
  - Even-numbered red cards in **Hearts (♥)** and **Diamonds (♦)**  
  - **Total red even cards = 10**  
  - **P(A ∩ B) = 10/52**  

#### **Union Probability (P(A ∪ B)) → Either A or B or both occur**  
$$
P(A ∪ B) = P(A) + P(B) - P(A ∩ B)
$$
where:  
- **P(A) = P(Red Card) = 26/52** (Marginal Probability)  
- **P(B) = P(Even Card) = 20/52** (Marginal Probability)  
- **P(A ∩ B) = P(Red Even Card) = 10/52**  

#### **Calculating Union Probability:**
$$
P(A ∪ B) = \frac{26}{52} + \frac{20}{52} - \frac{10}{52}
$$
$$
P(A ∪ B) = \frac{36}{52} = \frac{18}{26} = \frac{9}{13}
$$



#### **Summary of Probability Values**
| **Event**         | **Number of Cards** | **Probability** |
||--|-|
| **P(A) (Red Card)**  | 26                | $ \frac{26}{52} = \frac{1}{2} $ |
| **P(B) (Even Card)** | 20                | $ \frac{20}{52} = \frac{5}{13} $ |
| **P(A ∩ B) (Red Even Card)** | 10         | $ \frac{10}{52} = \frac{5}{26} $ |
| **P(A ∪ B) (Red OR Even Card)** | 36         | $ \frac{36}{52} = \frac{9}{13} $ |




#### **1. Mutually Exclusive (Disjoint) Events**

Two events are **mutually exclusive** (or disjoint) if they cannot occur at the same time. In other words, the occurrence of one event excludes the possibility of the other occurring simultaneously. Mathematically, for events A and B that are mutually exclusive,  
$$
P(A \cap B) = 0
$$

When events are mutually exclusive, the probability of either occurring (their union) is simply the sum of their individual probabilities:  
$$
P(A \cup B) = P(A) + P(B)
$$


#### **2. Example Using a Standard Deck of Cards**

#### **Scenario:**
Imagine you are drawing one card from a standard 52-card deck. Consider the following two events:

- **Event C (Red Card):** The card drawn is red (i.e., belongs to Hearts or Diamonds).  
  - **Total red cards:** 26 out of 52  
  - $$
    P(C) = \frac{26}{52} = \frac{1}{2}
    $$

- **Event D (Black Card):** The card drawn is black (i.e., belongs to Clubs or Spades).  
  - **Total black cards:** 26 out of 52  
  - $$
    P(D) = \frac{26}{52} = \frac{1}{2}
    $$

**Mutual Exclusivity:**  
Since a single card cannot be both red and black, events C and D are mutually exclusive. This means:
$$
P(C \cap D) = 0
$$

**Calculating the Union:**  
The probability of drawing a card that is either red or black (which in this case is a certainty) is:
$$
P(C \cup D) = P(C) + P(D) = \frac{1}{2} + \frac{1}{2} = 1
$$

#### **3. Contrast with Non-Mutually Exclusive Events**

For events that are **not** mutually exclusive (i.e., they can occur at the same time), the probability of either event occurring is calculated by:
$$
P(A \cup B) = P(A) + P(B) - P(A \cap B)
$$

#### **Earlier Example Recap: Joint Probability**
- **Event A (Red Card):** 26 cards  
- **Event B (Even-Numbered Card):** 20 cards  
- **Intersection (Red Even Card):** 10 cards  
  - Here, $P(A \cap B) = \frac{10}{52}$  
  - Because a card can be red and even numbered simultaneously, these events are **not** mutually exclusive.

- **Union Calculation for Non-Mutually Exclusive Events:**
  $$
  P(A \cup B) = \frac{26}{52} + \frac{20}{52} - \frac{10}{52} = \frac{36}{52} = \frac{9}{13}
  $$




#### **1. Independent Events**
**Definition:**  
Two events are **independent** if the outcome of one event does **not** affect the outcome of the other.  

Mathematically, if $ A $ and $ B $ are independent events, then:
$$
P(A \cap B) = P(A) \times P(B)
$$

#### **2. Example: Drawing Two Red Cards with Replacement**
**Given Events:**
- **Event $E_1$:** Drawing a red card on the first draw.
- **Event $E_2$:** Drawing a red card on the second draw.

A standard deck has **52 cards**, with **26 red cards**.

Since **replacement** is mentioned, after drawing a red card, we put it back into the deck before drawing again. This keeps the probabilities the same for both draws.

### **Probability Calculation:**
1. Probability of drawing a red card on the **first draw**:
   $$
   P(E_1) = \frac{26}{52} = \frac{1}{2}
   $$
2. Probability of drawing a red card on the **second draw** (since the first card was replaced, the probabilities remain unchanged):
   $$
   P(E_2) = \frac{26}{52} = \frac{1}{2}
   $$
3. Since these are **independent** events (replacement means the first draw does not affect the second), the probability of both occurring:
   $$
   P(E_1 \cap E_2) = P(E_1) \times P(E_2) = \frac{1}{2} \times \frac{1}{2} = \frac{1}{4} = 0.25
   $$

#### **3. Representation with a Sample Space**
With replacement, we consider all possible outcomes for two draws:

$$
\{ HH, TT, HT, TH \}
$$
Where:
- **H** = Red (Hearts/Diamonds)
- **T** = Black (Clubs/Spades)

Each outcome has a probability of **$ (1/2) \times (1/2) = 1/4 $ or 0.25**.

#### **4. Summary**
| **Concept**                 | **With Replacement** |
|--|-|
| **Definition**               | Events are independent when replacement occurs. |
| **Effect of Replacement**    | Keeps probabilities the same for each draw. |
| **Probability Formula**      | $ P(A \cap B) = P(A) \times P(B) $ |
| **Example (Red Card Twice)** | $ 26/52 \times 26/52 = 1/4 = 0.25 $ |
| **Possible Outcomes**        | {HH, TT, HT, TH} |

#### **Understanding Probability: With vs. Without Replacement**

This explanation breaks down the key concepts regarding **probability calculations with and without replacement** using a deck of cards.

#### **1. Probability Without Replacement**  
When drawing **without replacement**, the probability of drawing the second event **depends** on the first event.  

### **Example: Drawing Two Red Cards Without Replacement**  
- **First draw:** Probability of drawing a red card:  
  $$
  P(A) = \frac{26}{52} = \frac{1}{2}
  $$
- **Second draw:** Since one red card has already been removed, only **25 red cards remain** out of **51** total cards:  
  $$
  P(B | A) = \frac{25}{51}
  $$
- **Joint Probability (Both Red Cards):**  
  $$
  P(A \cap B) = P(A) \times P(B | A) = \frac{26}{52} \times \frac{25}{51} = \frac{650}{2652} \approx 0.245
  $$

**Key Formula for Dependent Events:**  
$$
P(A \cap B) = P(A) \times P(B | A)
$$
This accounts for the **changing probability** due to no replacement.

#### **2. Probability With Replacement**  
When drawing **with replacement**, the probability of drawing the second card remains **independent** of the first.

### **Example: Drawing Two Red Cards With Replacement**  
- **First draw:** $ P(A) = \frac{26}{52} $  
- **Second draw (since the first card is replaced, the probabilities don’t change):**  
  $$
  P(B) = \frac{26}{52}
  $$
- **Joint Probability (Both Red Cards):**  
  $$
  P(A \cap B) = P(A) \times P(B) = \frac{26}{52} \times \frac{26}{52} = \frac{676}{2704} \approx 0.25
  $$

**Key Formula for Independent Events:**  
$$
P(A \cap B) = P(A) \times P(B)
$$
This works **only if A and B are independent**, which is true **with replacement**.

#### **3. Key Differences: With vs. Without Replacement**

| **Feature**          | **With Replacement**                            | **Without Replacement**                         |
|-||--|
| **Definition**        | The first drawn card is placed back before drawing again. | The first drawn card is not placed back. |
| **Effect on Probabilities** | Probabilities stay the same for each draw. | Probabilities change after each draw. |
| **Are events independent?** | Yes, events are independent. | No, events are dependent. |
| **Formula for Joint Probability** | $ P(A \cap B) = P(A) \times P(B) $ | $ P(A \cap B) = P(A) \times P(B | A) $ |
| **Example (Drawing 2 Red Cards)** | $ \frac{26}{52} \times \frac{26}{52} = 0.25 $ | $ \frac{26}{52} \times \frac{25}{51} = 0.245 $ |

#### **4. Summary**  
- **Without Replacement** → **Probabilities change**, so we use **conditional probability** $ P(B | A) $.  
- **With Replacement** → **Probabilities remain the same**, so events are **independent**.  
- **Formula to Remember:**  
  - **Independent (With Replacement):** $ P(A \cap B) = P(A) P(B) $  
  - **Dependent (Without Replacement):** $ P(A \cap B) = P(A) P(B | A) $  

This breakdown clarifies the calculations and why replacement matters in probability!

#### **Mutually Exclusive vs. Independent Events**

| **Concept**                | **Mutually Exclusive (ME)**             | **Independent Events (IE)** |
|-|--|--|
| **Definition**              | Events that cannot happen together.    | Events that do not affect each other. |
| **Intersection Condition**  | $ P(A \cap B) = 0 $                 | $ P(A \cap B) = P(A) \times P(B) $ |
| **Example (Coin Flip)**     | Getting **heads** and **tails** at the same time → Impossible | Getting **heads** on one flip does not affect getting **heads** on the next. |
| **Formula for Probability** | $ P(A \cup B) = P(A) + P(B) $       | $ P(A \cap B) = P(A) P(B) $ |

#### **Key Takeaways**
- **Mutually Exclusive Events** **→ Cannot happen together** (No overlap).
- **Independent Events** **→ Can happen together**, and one event does not affect the probability of the other.
- If $ P(A \cap B) \neq P(A) \cdot P(B) $, then $ A $ and $ B $ are **dependent**.



Here's the revised version with all headings as **h4** and without horizontal lines:



#### **1. Union of Two Events (A ∪ B)**  
The probability of either event A or event B occurring is given by:  

$$ P(A \cup B) = P(A) + P(B) - P(A \cap B) $$

- **Case 1: Mutually Exclusive (ME) Events**  
  - If A and B are mutually exclusive (disjoint), they cannot occur together, meaning $ P(A \cap B) = 0 $.  
  - So, the formula simplifies to:  
    $$ P(A \cup B) = P(A) + P(B) $$  
- **Case 2: Not Mutually Exclusive (Not ME) Events**  
  - If A and B are not mutually exclusive, then they can happen together.  
  - In this case, we must subtract $ P(A \cap B) $ to avoid double counting.



#### **2. Intersection of Two Events (A ∩ B) - Joint Probability**  
The probability of both A and B occurring together is given by:

$$ P(A \cap B) = P(A) P(B|A) $$

- **Case 1: Mutually Exclusive (ME) Events**  
  - If A and B are mutually exclusive, they cannot occur together.  
  - This means $ P(A \cap B) = 0 $.  
- **Case 2: Not Mutually Exclusive (Not ME) Events**  
  - Here, events A and B can occur together, and we use the conditional probability formula.



#### **3. Dependent vs. Independent Events**  
Events can either be dependent or independent:  

#### **Independent Events**  
- A and B are independent if the occurrence of one does not affect the probability of the other.  
- Mathematically,  

  $$ P(A \cap B) = P(A) P(B) $$

  This means:

  $$ P(B|A) = P(B) $$

#### **Dependent Events**  
- A and B are dependent if the occurrence of A changes the probability of B.  
- In this case:

  $$ P(A \cap B) = P(A) P(B|A) $$

  and

  $$ P(B|A) \neq P(B) $$



#### **4. Conditional Probability $ P(A|B) $**  
Conditional probability measures the likelihood of A occurring, given that B has already occurred. The formula is:

$$ P(A|B) = \frac{P(A \cap B)}{P(B)} $$

- **Case 1: Mutually Exclusive (ME) Events**  
  - If A and B are mutually exclusive, then $ P(A|B) = 0 $ because they cannot happen together.  
- **Case 2: Not Mutually Exclusive (Not ME) Events**  
  - Here, we use the above formula to calculate conditional probability.



#### **5. Law of Total Probability**  
This law helps in calculating probabilities when there are multiple cases. It states:

$$ P(B) = P(A \cap B) + P(A^c \cap B) $$

This means we break the probability into two parts:  
1. The probability of B occurring when A happens.  
2. The probability of B occurring when A does not happen (A complement).  

This is useful when dealing with multiple cases and conditions.



#### **Summary of Key Differences**  
- **Mutually Exclusive (ME) Events** → $ P(A \cap B) = 0 $  
- **Independent Events** → $ P(A \cap B) = P(A) P(B) $  
- **Dependent Events** → $ P(A \cap B) = P(A) P(B|A) $  
- **Conditional Probability** helps determine $ P(A) $ given $ B $ has happened.  
- **Total Probability** helps find overall probability by breaking it into cases.  




#### **1. Union of Two Events $P(A \cup B)$**  

#### **Example:**  
A survey finds that 40% of people watch Netflix, 30% watch Amazon Prime, and 15% watch both. What is the probability that a randomly selected person watches either Netflix or Amazon Prime?  

#### **Solution:**  
We use the formula:  

$$ P(A \cup B) = P(A) + P(B) - P(A \cap B) $$

Where:  
- $ P(A) = 0.40 $ (Probability of watching Netflix)  
- $ P(B) = 0.30 $ (Probability of watching Amazon Prime)  
- $ P(A \cap B) = 0.15 $ (Probability of watching both)  

$$ P(A \cup B) = 0.40 + 0.30 - 0.15 $$

$$ = 0.55 $$

#### **Explanation:**  
- We add both probabilities since we need "either" event.  
- We subtract $ P(A \cap B) $ because those watching both were counted twice.



#### **2. Intersection of Two Events $ P(A \cap B) $ - Joint Probability**  

#### **Example:**  
A company has a 60% chance of getting Project A and a 50% chance of getting Project B. If the probability of getting both projects is 30%, what is the probability that the company gets at least one project?  

#### **Solution:**  
We use the same formula as before:

$$ P(A \cup B) = P(A) + P(B) - P(A \cap B) $$

Given:  
- $ P(A) = 0.60 $  
- $ P(B) = 0.50 $  
- $ P(A \cap B) = 0.30 $  

$$ P(A \cup B) = 0.60 + 0.50 - 0.30 $$

$$ = 0.80 $$

So, there is an 80% chance that the company gets at least one project.

#### **Explanation:**  
- $ P(A \cap B) $ represents the probability that both projects happen together.  
- We remove it to avoid double counting.  



#### **3. Dependent vs. Independent Events**  

#### **Example:**  
A bag contains 4 red and 6 blue balls. You draw two balls **without replacement**. What is the probability that both are red?  

#### **Solution:**  
#### **Step 1: Probability of drawing the first red ball**  
There are 4 red balls out of 10:

$$ P(A) = \frac{4}{10} = 0.4 $$

#### **Step 2: Probability of drawing the second red ball (dependent event)**  
Since we do not replace the first ball, only 3 red balls remain out of 9:

$$ P(B|A) = \frac{3}{9} = \frac{1}{3} $$

#### **Step 3: Multiply for dependent events**  
Since drawing the second red ball depends on the first:

$$ P(A \cap B) = P(A) P(B|A) $$

$$ = 0.4 \times \frac{1}{3} = 0.1333 $$

Thus, the probability of drawing two red balls is **0.1333 (13.33%)**.

#### **Explanation:**  
- The second probability changes because the first ball is **not replaced**, making the events **dependent**.  
- If we replaced the ball, the probability would stay the same, making the events **independent**.  



#### **4. Conditional Probability $ P(A|B) $**  

#### **Example:**  
In a factory, 70% of the products pass quality control (A). If a product passes, the probability that it is from Machine X (B) is 80%. What is the probability that a product is from Machine X given that it passed quality control?  

#### **Solution:**  
The formula is:

$$ P(A|B) = \frac{P(A \cap B)}{P(B)} $$

Given:  
- $ P(A) = 0.70 $ (passing quality control)  
- $ P(B|A) = 0.80 $ (from Machine X if passed)  
- We need $ P(B) $. Using total probability:

$$ P(A) = P(A|B) P(B) + P(A|\bar{B}) P(\bar{B}) $$

- Assume $ P(B) = 0.50 $ (50% of products from Machine X).  
- $ P(\bar{B}) = 0.50 $, and $ P(A|\bar{B}) = 0.60 $ (passing if **not** from Machine X).  

$$ P(A) = (0.80)(0.50) + (0.60)(0.50) $$

$$ = 0.40 + 0.30 = 0.70 $$

Now, we calculate:

$$ P(B|A) = \frac{P(A \cap B)}{P(A)} $$

$$ = \frac{0.40}{0.70} = 0.57 $$

So, the probability that a product is from Machine X given that it passed quality control is **0.57 (57%)**.

#### **Explanation:**  
- We use conditional probability because we know the event "passed quality control" happened.  
- We applied the **law of total probability** to find $ P(A) $.  



#### **5. Law of Total Probability**  

#### **Example:**  
A software company has two departments:  
- **Development** (60% of employees)  
- **Testing** (40% of employees)  

The probability of getting a promotion is:  
- **50% in Development**  
- **30% in Testing**  

What is the overall probability that a randomly chosen employee gets a promotion?  

#### **Solution:**  
The law of total probability states:

$$ P(A) = P(A|B) P(B) + P(A|\bar{B}) P(\bar{B}) $$

Where:  
- $ P(B) = 0.60 $ (Development)  
- $ P(\bar{B}) = 0.40 $ (Testing)  
- $ P(A|B) = 0.50 $ (Promotion in Development)  
- $ P(A|\bar{B}) = 0.30 $ (

Promotion in Testing)  

Applying the formula:

$$ P(A) = (0.50)(0.60) + (0.30)(0.40) $$

$$ = 0.30 + 0.12 = 0.42 $$

Thus, the overall probability of an employee getting a promotion is **42%**.

#### **Explanation:**  
- We used the **law of total probability** because the promotion depends on which department an employee belongs to.  
- We weighted the probabilities according to department sizes.

 

## **Final Summary of Concepts with Examples:**  
| **Concept**               | **Example Summary** |
|---------------------------|---------------------|
| **Union (A ∪ B)**         | Netflix & Amazon Prime subscribers, using $P(A) + P(B) - P(A \cap B)$. |
| **Intersection (A ∩ B)**  | Getting at least one project in a company, using joint probability. |
| **Dependent Events**      | Drawing two red balls **without** replacement, adjusting probability dynamically. |
| **Conditional Probability** | Factory machine producing a quality product, calculated using $P(A|B)$. |
| **Law of Total Probability** | Company promotion rates across departments, weighted calculation. |



#### 1. Probability of (Ace OR King)

- **Formula Used**:  
  $$
  P(A \text{ or } K) = P(A) + P(K)
  $$

- **Calculation**:  
  $$
  P(A) = \frac{4}{52}, \quad P(K) = \frac{4}{52} \Rightarrow P(A \text{ or } K) = \frac{4}{52} + \frac{4}{52} = \frac{8}{52}
  $$

- **Why This Formula**:  
  This formula is used because the events "Ace" and "King" are mutually exclusive (they cannot occur at the same time), so we add their probabilities directly.


#### 2. Probability of (Heart OR Spade)

- **Formula Used**:  
  $$
  P(H \text{ or } S) = P(H) + P(S)
  $$

- **Calculation**:  
  $$
  P(H) = \frac{13}{52}, \quad P(S) = \frac{13}{52} \Rightarrow P(H \text{ or } S) = \frac{13}{52} + \frac{13}{52} = \frac{26}{52}
  $$

- **Why This Formula**:  
  Similar to the previous example, the events "Heart" and "Spade" are mutually exclusive. Their probabilities are added together to determine the total probability.


#### 3. Probability of (Queen OR Black)

- **Formula Used**:  
  $$
  P(Q \text{ or } B) = P(Q) + P(B) - P(Q \cap B)
  $$

- **Calculation**:  
  $$
  P(Q) = \frac{4}{52}, \quad P(B) = \frac{26}{52}, \quad P(Q \cap B) = \frac{2}{52} \;(\text{Queens in black suits}) 
  $$
  $$
  P(Q \text{ or } B) = \frac{4}{52} + \frac{26}{52} - \frac{2}{52} = \frac{28}{52}
  $$

- **Why This Formula**:  
  In this case, the events "Queen" and "Black" are not mutually exclusive because there are Queens that are also Black (the Queen of Spades and the Queen of Clubs). Thus, we subtract the intersection to avoid double counting.



#### **4. Probability of (King OR Heart)**

- **Formula Used**:  
  $$
  P(K \text{ or } H) = P(K) + P(H) - P(K \cap H)
  $$

- **Calculation**:  
  $$
  P(K) = \frac{4}{52}, \quad P(H) = \frac{13}{52}, \quad P(K \cap H) = \frac{1}{52} \;(\text{King of Hearts}) 
  $$
  $$
  P(K \text{ or } H) = \frac{4}{52} + \frac{13}{52} - \frac{1}{52} = \frac{16}{52}
  $$

- **Why This Formula**:  
  Again, the events "King" and "Heart" are not mutually exclusive due to the King of Hearts. The intersection is subtracted to ensure that we do not count this double.




#### 1. Two Coins Tossed

- **Problem**: Find the probability of getting HT (Heads on the first coin and Tails on the second coin).

- **Formula Used**:  
  $$
  P(HT) = P(H) \cdot P(T)
  $$

- **Calculation**:  
  $$
  P(H) = \frac{1}{2}, \quad P(T) = \frac{1}{2} \Rightarrow P(HT) = \frac{1}{2} \cdot \frac{1}{2} = \frac{1}{4}
  $$

- **Why This Formula**:  
  Since the tosses are independent events, we multiply the probabilities of the individual events (getting Heads and Tails) to find the combined probability.



#### 2. Two Decks of Cards

- **Problem**: Find the probability of drawing an Ace and a King.

- **Formula Used**:  
  $$
  P(Ace, King) = P(A) \cdot P(K)
  $$

- **Calculation**:  
  $$
  P(A) = \frac{4}{52}, \quad P(K) = \frac{4}{52} \Rightarrow P(Ace, King) = \frac{4}{52} \cdot \frac{4}{52} = \frac{16}{2704}
  $$

- **Why This Formula**:  
  The formula is applied here as each event (drawing an Ace and then a King) is independent when drawing from two separate decks.



#### 3. One Deck for Ace and King

- **Problem**: Find the probability of drawing an Ace and then a King, considering the Ace has already been drawn.

- **Formula Used**:  
  $$
  P(Ace, King) = P(Ace) \cdot P(King|Ace)
  $$

- **Calculation**:  
  $$
  P(Ace) = \frac{4}{52}, \quad P(King|Ace) = \frac{4}{51} \Rightarrow P(Ace, King) = \frac{4}{52} \cdot \frac{4}{51}
  $$

- **Why This Formula**:  
  This formula accounts for the fact that the events are dependent; the probability of drawing a King changes after an Ace has been drawn (one card is removed from the deck).




### Example: Probability of Defective Items

#### Problem Statement
A manufacturer produces items with a defect rate of 2%. We want to find the probability that at least one of four inspected items is defective.


#### 1. Relevant Probabilities

- **Given Probability of Defective Item**:
  $$
  P(\text{defective}) = 0.02
  $$
  
- **Probability of Non-defective Item**:
  $$
  P(\text{not defective}) = 1 - P(\text{defective}) = 0.98
  $$


#### 2. Formula Used
To find the probability of at least one defective item among four inspected, we can use the complement rule. The formula can be expressed as:

$$
P(\text{at least one defective}) = 1 - P(\text{none defective})
$$

Where:
$$
P(\text{none defective}) = P(\text{not defective})^n
$$
with $ n $ being the number of items inspected.

For this scenario:
$$
P(\text{none defective}) = P(\text{not defective})^4 = (0.98)^4
$$


#### 3. Calculation Steps

1. **Calculate Probability of None Defective**:
   $$
   P(\text{none defective}) = (0.98)^4
   $$

   **Calculation**:
   $$
   (0.98)^4 \approx 0.92236816
   $$

2. **Calculate Probability of At Least One Defective**:
   $$
   P(\text{at least one defective}) = 1 - P(\text{none defective}) = 1 - 0.92236816 \approx 0.07763184
   $$








### Example: Probability of Working Machines
A casino has two types of machines: type X and type Y. 

- Machines of type **X** have a 99% success rate (work perfectly).
- Machines of type **Y** have a 95% success rate.
- 60% of the machines are from type X, and 40% are from type Y.

We want to find the probability that a randomly selected machine works perfectly.


- **Events**:
  - $ A $: The machine works perfectly.
  - $ B $: The machine is of type X.
  - $ C $: The machine is of type Y.



- $ P(B) = 0.6 $ (Probability that a machine is type X)
- $ P(C) = 0.4 $ (Probability that a machine is type Y)
- $ P(A|B) = 0.99 $ (Probability that a machine from type X works perfectly)
- $ P(A|C) = 0.95 $ (Probability that a machine from type Y works perfectly)


To find the total probability that a randomly selected machine works perfectly, we use the **Law of Total Probability**:

$$
P(A) = P(A|B) \cdot P(B) + P(A|C) \cdot P(C)
$$



1. **Calculate Contribution from Machines of Type X**:
   $$
   P(A|B) \cdot P(B) = 0.99 \cdot 0.6 = 0.594
   $$

2. **Calculate Contribution from Machines of Type Y**:
   $$
   P(A|C) \cdot P(C) = 0.95 \cdot 0.4 = 0.38
   $$

3. **Total Probability of a Machine Working Perfectly**:
   $$
   P(A) = 0.594 + 0.38 = 0.974
   $$

### 3. Why This Formula?

We use the **Law of Total Probability** because we want to find the overall probability that a machine works perfectly, breaking it down into contributions from the different machine types. Each type has its own success rate and proportion, and this method allows us to accurately combine these relevant probabilities to get the total probability.



